.github/workflows/ci.yml

```diff
@@ -133,8 +135,20 @@ jobs:
           pip install -U -c constraints.txt -r requirements-dev.txt
       - name: Run integration tests
         run: make integration-test-coverage
-      - name: Upload integration test coverage report
-        uses: actions/upload-artifact@v2
+      - name: Report coverage to coveralls.io
+        uses: coverallsapp/github-action@1.1.3
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          flag-name: integration-tests_${{ matrix.environment }}
+          parallel: true
+          path-to-lcov: coverage.lcov
+  tests-finished:
```

Comment:
Could you please add a name for this section so it looks good in [this](https://github.com/daka1510/qiskit-ibm-runtime/actions/runs/1844766846) view?

Reply:
See https://github.com/Qiskit/qiskit-ibm-runtime/pull/144/commits/c0b416243f43a47be2c2009a042820d49a1831a7